ApollosProject/apollos-apps
# feat: Community Stream Activity Feed

## Motivation
This PR updates the Community Single to include an onScroll animation in preparation for there to be the addition of a FlatList for the Stream Activity Feed. While the feed is not complete, it is included in this PR as a first round and more PR's will come out to fully support the posting/viewing of each post-type.

## Considerations
There are a bunch of test activities inside of one of the Feeds inside of Stream, so this PR just hardcodes that ID to make testing easier. Once we're able to actually post from the front end, we'll update this to use the community I'm actually looking at.

## Testing
1. Make sure that you have the latest from the API (I _think_ you can use stage, but I did everything locally
2. Set up your dB Config to include `STREAM.API_KEY`, `STREAM.API_KEY_SECRET`, `STREAM.APP_ID`
3. Set up your mobile app env to include `STREAM_FEED_API_KEY` and `STREAM_FEED_APP_ID`
4. Launch the mobile app and navigate to the Community Single screen

| iOS | iPadOS | Android |
| --- | --- | --- |
| ![CleanShot 2022-09-22 at 09 46 24](https://user-images.githubusercontent.com/45076058/191764155-a15ac5cf-2ca3-4914-8029-39d0219683b9.png) | ![CleanShot 2022-09-22 at 09 49 17](https://user-images.githubusercontent.com/45076058/191764831-c2cd170a-e0b9-4223-a600-2c03be835ce4.png) | ![Screenshot_20220922-095400](https://user-images.githubusercontent.com/45076058/191766576-82397732-73eb-4797-8a60-f7bf41f95cb7.png) |
